app/src/main/java/com/noalecohen/dispatcher/FavoritesFragment.kt

```diff
@@ -0,0 +1,33 @@
+package com.noalecohen.dispatcher
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.noalecohen.dispatcher.databinding.FragmentFavoritesBinding
+import model.BaseFragment
+
+class FavoritesFragment : BaseFragment() {
+    private lateinit var binding: FragmentFavoritesBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        binding = FragmentFavoritesBinding.inflate(inflater, container, false)
+        displayAuthor()
+        return binding.root
+    }
+
+    private fun displayAuthor() {
+        var authors = ""
+        for(article in articleList) {
+            if(article.Author != null) {
+                authors += article.Author
+            }
+            authors += "\n\n"
+        }
+        binding.favoritesText.text = authors
```

Comment:
same comments as in AccountFragment

Reply:
I guess the answer is 'no'... Can you help me please?

Reply:
You guessed right. Kotlin is a high level language that allows writing code which is highly readable. Please try to think of an implementation which is more readable. A tip could be to try to create it in a descriptive way and not a creative one. Good luck.

Reply:
I've tried to implement this with a listView, but the app crashed and I don't understand why...